### PR TITLE
Minor sample app fixes

### PR DESCRIPTION
**Pull request by yihezkel**
### Fixed
Minor sample app fixes

### CODE REVIEW

File: quickstart/pom.xml [hunk 1]
```diff
@@ -154,9 +154,9 @@
             <artifactId>opentelemetry-exporter-otlp</artifactId>
         </dependency>
         <dependency>
-            <groupId>io.opentelemetry</groupId>
+            <groupId>io.opentelemetry.semconv</groupId>
             <artifactId>opentelemetry-semconv</artifactId>
-            <version>1.30.1-alpha</version>
+            <version>1.23.1-alpha</version>
```
Comment (AsafMah) [inline quickstart/pom.xml:159]: Why?

**Review by AsafMah — COMMENTED** [review]
why  the pom change?

File: quickstart/pom.xml (hunk 1 above)
Comment (yihezkel) [reply quickstart/pom.xml:159]: https://javadoc.io/static/io.opentelemetry/opentelemetry-semconv/1.30.1-alpha/deprecated-list.html

Comment (yihezkel) [reply quickstart/pom.xml:159]: Btw, this is not a downgrade:
https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-semconv
https://mvnrepository.com/artifact/io.opentelemetry.semconv/opentelemetry-semconv

**Review by AsafMah — APPROVED** [review]

**Pull request merged**